### DISCUSSION TITLE
should also add the tab for maximum streak

### DISCUSSION THREAD

**Discussion by ajeetgupta80** [monkeytypegame/monkeytype#3999] [Ideas] [answered]
like now it only have the feature of showing current streak but should also add a feature for showing the maximum streak at anytime obtain by an user

**Comment by grigoriynikolaev** [comment]
You can mouse over your current streak and it will display your longest one.

**Reply by ajeetgupta80** [reply, marked as answer]
ya but if you dont have current streak then where you will hover mouse i mean they should have a dedicated longest streak tab or something like that
